dist/tags/framework/debugger.html

```diff
@@ -407,8 +452,33 @@
   var $root = document.getElementById('__tagsmith_debugger');
   var $open = $root.querySelector('.__tagsmith_debugger_open');
   var $container = $root.querySelector('.__tagsmith_debugger_container');
+  var $testSetup = $root.querySelector('#__tagsmith_debugger_test_setup');
   var $close = $root.querySelector('.__tagsmith_debugger_close');
 
+  var testSetupLogger;
```

Comment:
It looks like it have the same functions with same name, is it possible to integrate the function so that either one or both calls the function?

https://github.com/google-marketing-solutions/tagsmith/pull/1/files#diff-168880041785d5b921d32a074f44c3a42a6db13f7e799396a712916a07450836R155

Reply:
Files in `dist` is built from `src` thus it's expected to be duplicated.
Will add a note in [contributing.md](https://github.com/google-marketing-solutions/tagsmith/blob/main/docs/contributing.md) later.